### PR TITLE
BXMSPROD-69: Archive kie-docs artifacts

### PR DESCRIPTION
**Pull request by jomarko**
@mbiarnes @pszubiak would you mind to review?

### CODE REVIEW

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/PrVerificationJob.groovy [hunk 1]
```diff
@@ -197,4 +203,35 @@ class PrVerificationJob extends BasicJob {
             }
         }
     }
+
```
Comment (pszubiak) [inline job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/PrVerificationJob.groovy]: Thank you for PR! Do we need this?

Comment (jomarko) [reply job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/templates/PrVerificationJob.groovy]: Thanks for spotting, I will remove.

**Comment by jomarko** [comment]
@pszubiak I updated the PR, please have a look and let me know.

**Comment by jomarko** [comment]
@pszubiak I named the argument to be consistent as discussed on slack

**Pull request merged**